mlrun/config.py

```diff
@@ -567,18 +567,18 @@
         },
         "application_stream_args": {
             "v3io": {
-                "shard_count": 1,
+                "shard_count": 8,
                 "retention_period_hours": 24,
-                "num_workers": 1,
-                "min_replicas": 1,
-                "max_replicas": 1,
+                "num_workers": 4,
+                "min_replicas": 8,
+                "max_replicas": 8,
             },
             "kafka": {
-                "partition_count": 1,
+                "partition_count": 16,
                 "replication_factor": 1,
-                "num_workers": 1,
+                "num_workers": 4,
                 "min_replicas": 1,
-                "max_replicas": 1,
+                "max_replicas": 8,
```

Comment:
If I understand the requirement right... I think this should be  the config
```suggestion
                "shard_count": 4,
                "retention_period_hours": 24,
                "num_workers": 4,
                "min_replicas": 1,
                "max_replicas": 1,
            },
            "kafka": {
                "partition_count": 4,
                "replication_factor": 1,
                "num_workers": 4,
                "min_replicas": 1,
                "max_replicas": 1,
```

Reply:
Thanks! Will change as you suggest